_posts/2020-11-12-include-anything-in-grafana.md

```diff
@@ -0,0 +1,36 @@
+---
+title: How to include anything in Grafana
+layout: post
+author: Sam Zadworny
+email: zado1984@gmail.com
+---
+
+## Introduction
+
```

Comment:
Nice! Here  at the very begining I'd add something about "many people have added influxdb and grafana to their setup, but there's this problem...."

The first time Grafana is mentioned it could be a link so that people can get more info on it.

Reply:
Done. Let me know if it's acceptable. I also added another screenshot of a change required in the Grafana .ini file. Thanks.